avm/res/synapse/workspace/big-data-pools/main.bicep

```
@@ -0,0 +1,282 @@
+metadata name = 'Synapse Workspaces Big Data Pools'
+metadata description = 'This module deploys Synapse Workspaces Big Data Pools.'
+
+// ================ //
+// Parameters       //
+// ================ //
+
+@description('Required. The name of the parent Synapse Workspace.')
```

For parents we should use the following
```suggestion
@description('Conditional. The name of the parent Synapse Workspace. Required if the template is used in a standalone deployment.')')
```

---

Updated